Improve SUSE Manger 5.0 proxy installation and migration (#3158)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration-suma.adoc

= Proxy Migration

In {productname} 4.3, the proxy can be deployed using three different methods: RPM based, containerized running on podman or k3s.

In {productname} {productnumber}, management of the containerized proxy running with podman was re-designed and made simpler with the [command]``mgrpxy`` tool.
At the same time, RPM based support was removed, and only the containerized version running with podman or k3s is supported.

> This section will help you migrate from 4.3 proxy using the new [command]``mgrpxy`` tool.

Reviewer: ```suggestion
This section will describes migrating from Proxy 4.3 using the new [command]``mgrpxy`` tool.
```

[IMPORTANT]
====

> An in-place migration from {productname} 4.3 to 5.0 is unsupported due to the HostOS change from {sles} 15 SP4 to {micro} {microversion}

Reviewer: ```suggestion
An in-place migration from {productname} 4.3 to 5.0 is unsupported due to the HostOS change from {sles} 15 SP4 to {micro} {microversion}.
```

== Deploy a new {productname} proxy

> As said before, in place migration is not supported, this means users must deploy a new {productname} proxy with a new FQDN.

Reviewer: ```suggestion
As in place migration is not supported, the usersmust deploy a new {productname} proxy with a new FQDN.
```

> * For more on how to install {productname} proxy, see xref:container-deployment/suma/server-deployment-vm-suma.adoc[Deploy Proxy as a Virtual Machine].

Reviewer: ```suggestion
* For more information on how to install {productname} proxy, see xref:container-deployment/suma/server-deployment-vm-suma.adoc[Deploy Proxy as a Virtual Machine].
```

== Migrate minions to the new proxy

[IMPORTANT]
====

> The new proxy is already deployed and fully functional.

Reviewer 2: ```suggestion
Before migrating the minions, ensure that the new proxy is already deployed and fully functional.
```

====

.Task: Migrate minion between proxies
. Log in to the {productname} Server {webui}.

> . Select menu:Systems[Systems List] from the left navigation.

Reviewer 3: . From the left navigation, select menu:Systems[Systems List].

> . Navegate to the Old 4.3 proxy page, and enter the tab menu "Proxy".

Reviewer 4: ```suggestion
. Navigate to the old 4.3 proxy page, and click the [guimenu]``Proxy`` tab.
```

. Select all system to "SSM".

> . Select menu:Systems[System Set Manager] from the left navigation.

Reviewer 3: . From the left navigation, select menu:Systems[System Set Manager].

> . Select the sub-menu "Misc" -> "Proxy"

Reviewer 4: ```suggestion
. Select the sub-menu menu:Misc[Proxy].
```

. From the drop down select the new proxy to migrate to.

> . Select "change proxy".

Reviewer 4: ```suggestion
. Click btn:[change proxy].
```

> . Select "change proxy".

Reviewer 3: . Click on btn:[Change Proxy].

> . After this action minions will be migrated to the new proxy.

Reviewer 3: After this action, minions will be migrated to the new proxy.

> . User can check the schedule progress. Verify if all sucessed

Reviewer 4: ```suggestion
. User can schedule the action and check the progress.
  Verify if all succeeded.
```

Author: the action is schedule when you click on change button. In here what we want to say is that the user can verify the progress by click on the link in the banner that will show up, saying the action was scehdule.

> . User can check the schedule progress. Verify if all sucessed

Reviewer 3: . You can check the schedule progress to verify if all systems were successfully migrated.

> After a few minutes the machines will start to show up the new connection path. When all machines have the connection path under the new proxy, the 4.3 proxy machine is note needed anymore, and can be removed.

Reviewer 4: ```suggestion
After a few minutes, the machines will start to show up the new connection path.
When all machines have the connection path under the new proxy, the old 4.3 proxy machine is not needed anymore and can be removed.
```
